pkg/config/config.go

```diff
@@ -43,13 +43,24 @@ func DefineConfiguration(
 	v.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
 	v.AutomaticEnv()
 
+	confschema := schema
+	confschema.Fields = append(field.DefaultFields, confschema.Fields...)
+	// Ensure unique fields
+	uniqueFields := make(map[string]field.SchemaField)
+	for _, f := range confschema.Fields {
+		uniqueFields[f.FieldName] = f
```

Comment:
should this panic or otherwie fail?

Reply:
It should panic these are schema definitions we wrote

Reply:
It fails later when we attempt to add a duplicate flag actually

Reply:
why should this panic tho?

Reply:
duplicate flags was my worry